mmaction/models/recognizers/base.py

```diff
@@ -22,20 +22,26 @@ class BaseRecognizer(nn.Module, metaclass=ABCMeta):
 
     Args:
         backbone (dict): Backbone modules to extract feature.
-        cls_head (dict): Classification head to process feature.
+        cls_head (dict | None): Classification head to process feature.
+            Default: None.
+        feature_extraction (bool): Perform feature extraction instead of
+            recognition. Default: False.
+        neck (dict | None): Neck for feature fusion. Default: None.
         train_cfg (dict | None): Config for training. Default: None.
         test_cfg (dict | None): Config for testing. Default: None.
     """
 
     def __init__(self,
                  backbone,
-                 cls_head,
+                 cls_head=None,
+                 feature_extraction=False,
                  neck=None,
                  train_cfg=None,
                  test_cfg=None):
         super().__init__()
         # record the source of the backbone
         self.backbone_from = 'mmaction2'
+        self.feature_extraction = feature_extraction
```

Comment:
How about put `feature_extraction` in `test_cfg`?

Reply:
Good Idea.